M2/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ cmake_minimum_required(VERSION 3.15...3.19)
 cmake_policy(VERSION 3.14)
 
 ## Use the C++14 standard
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Nice! Whenever you get a chance, could you update the commented line above too?

Reply:
Fixed in https://github.com/Macaulay2/M2/pull/1861/commits/9b2791241e2a5762d6faae940ca695b841bf9a77.